Payments/Operation/05_ManageConfirmations.md

```diff
@@ -0,0 +1,123 @@
+[!!LIST (Confirmations](../UserInterface/06a_ListConfirmations.md)
+
+
+# Manage confirmations
+
+The *Confirmation* menu entry provides you with a view on all references to refunds that has been created. Before your company sends back the money to the customer, you have to check the refunds with this menu entry. This service is supported by the most payment service providers.
+> [Info] If a workflow decides that you have to sent back money to the customer, you have here the possibility to check the refunds before money is returned.   
```

Comment:
Vielleicht nicht als Info, sondern zusammen mit dem Absatz oben, vielleicht sogar in Verbindung mit dem zweiten Satz?

Reply:
done